README.md

```diff
@@ -34,17 +34,17 @@ Here are two prompts, each for executing different analysis processes.
 ```
 1st Prompt: Initialize the rule instance with the rule path set to "rule.json"
 ```
-Used Quark Script APIs/Tools that LLM used:  `Rule()`
```

Comment:
In quarkScriptAgent.py,  the tool `Rule` is named `loadRule`. I suggest keeping the name consistent.